app/models/group.rb

```diff
@@ -0,0 +1,3 @@
+class Group < ApplicationRecord
+  validates_presence_of :name, :responsable_name, :responsable_contact, :from_date, :to_date
```

Comment:
Is this line too long? Did rubocop not complain??

Reply:
Actually it did. I tried to put on multiline but I got errors. I just find out why: I didn't leave :name on the first line. 

Done in fixup 1d5a0f2

Reply:
Also, the new validation syntax is using: `validates :name, presence: true` instead of `validates_presence_of`
Try something like:
```
validates(
  :from_date,
  :name,
  :responsable_name,
  :responsable_contact,
  :to_date,
  presence: true,
)
```